crime-applications-adaptor/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/crimeapply/OtherIncomeMapperTest.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.justice.laa.crime.applications.adaptor.mapper.crimeapply;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.laa.crime.applications.adaptor.enums.OtherIncomeDetails;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.common.AssessmentDetail;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.general.OtherIncome;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class OtherIncomeMapperTest {
+
+    private OtherIncomeMapper otherIncomeMapper;
+    private final String OTHER_INCOME_DETAILS = "Here are some details.";
+    private final int AMOUNT = 100;
```

Comment:
Constants should also have the static keyword

Reply:
done